Utils/HandlerUtil.py

```diff
@@ -148,15 +149,21 @@ def log(self, message):
     def error(self, message):
         self._error(self._get_log_prefix() + message)
 
+    @staticmethod
```

Comment:
Looks good. Is there a way you can test this code in the unit-tests? (you can create a new test file if you don't want the baggage of earlier tests, which do not run).

Reply:
And we can sync tomorrow how you can use DCR to the E2E scenario tests.

Reply:
Added unit test for static method.